packages/blocks/src/store/test/reducer.js

```diff
@@ -108,12 +108,12 @@ describe( 'blockStyles', () => {
 		expect( blockStyles( undefined, {} ) ).toEqual( {} );
 	} );
 
-	it( 'should add a new block styles', () => {
+	it( 'should add new block styles for a single block type', () => {
 		const original = deepFreeze( {} );
 
 		let state = blockStyles( original, {
 			type: 'ADD_BLOCK_STYLES',
-			blockName,
+			blockNames: [ blockName ],
```

Comment:
```suggestion
			blockNames: blockName,
```
I think we should test this with a string, to check it works both ways.

Reply:
Well this breaks the tests, so I guess we need to be sure to cast it to an array!

Reply:
Thanks for taking a look at this part of the PR 👍 

I'm not sure we need to cover the possibility of someone dispatching their own action object for this. If they use `addBlockStyles` to create the action that will handle the casting any string given to an array.

Maybe I could actually use `addBlockStyles` within the tests here to sort of cover your suggestion?

Reply:
I've switched the tests to use `addBlockStyles` which illustrates the use case with a single block name as a string.